op-service/signer/transaction_args.go

```diff
@@ -129,38 +139,40 @@ func (args *TransactionArgs) ToTransactionData() (types.TxData, error) {
 	if args.AccessList != nil {
 		al = *args.AccessList
 	}
-	if len(args.BlobVersionedHashes) > 0 {
-		chainID, overflow := uint256.FromBig((*big.Int)(args.ChainID))
-		if overflow {
-			return nil, fmt.Errorf("chainID %s too large for blob tx", args.ChainID)
-		}
-		maxFeePerGas, overflow := uint256.FromBig((*big.Int)(args.MaxFeePerGas))
-		if overflow {
-			return nil, fmt.Errorf("maxFeePerGas %s too large for blob tx", args.MaxFeePerGas)
-		}
-		maxPriorityFeePerGas, overflow := uint256.FromBig((*big.Int)(args.MaxPriorityFeePerGas))
-		if overflow {
-			return nil, fmt.Errorf("maxPriorityFeePerGas %s too large for blob tx", args.MaxPriorityFeePerGas)
```

Comment:
Moved all this conversion code to `convertToUint256` helper method below to reduce duplicate code